.env.example

```diff
@@ -0,0 +1 @@
+NODE_ENV=development
```

Comment:
Should we make `development` the default and change the conditions to detect the `production` environment instead? Then we wouldn't need the env file locally.

Reply:
Done. That's a nice catch since there are no other envs we can default to dev.